windows/src/engine/keyman32/kmprocess.cpp

```diff
@@ -770,14 +770,14 @@ BOOL ContextMatch(LPKEY kkp)
 
         temp = xstrchr(s->dpString, q);
 
-        /*SendDebugMessageFormat(state.msg.hwnd, sdmKeyboard ,kkp->Line, "ContextMatch: CODE_ANY [%x %x %x %x %x %x %x %x %x %x] [%x %x %x] %d", 
-          s->dpString[0], s->dpString[1], s->dpString[2], 
-          s->dpString[3], s->dpString[4], s->dpString[5], 
-          s->dpString[6], s->dpString[7], s->dpString[8], 
+        /*SendDebugMessageFormat(state.msg.hwnd, sdmKeyboard ,kkp->Line, "ContextMatch: CODE_ANY [%x %x %x %x %x %x %x %x %x %x] [%x %x %x] %d",
+          s->dpString[0], s->dpString[1], s->dpString[2],
+          s->dpString[3], s->dpString[4], s->dpString[5],
+          s->dpString[6], s->dpString[7], s->dpString[8],
           s->dpString[9],
           q[0], q[1], q[2],
-          (temp ? (int)(temp-s->dpString) : 0));*/
-        
+          (temp ? (INT_PTR)(temp-s->dpString) : 0));*/
```

Comment:
Yeah, I know this code is currently commented out ... but I didn't want to skip it nor delete it at this point in time.